meson.build

```diff
@@ -0,0 +1,88 @@
+project('sleepdart', 'c', default_options: ['buildtype=release'])
+
+release_args = ['-ffast-math', '-flto', '-fwhole-program']
+if get_option('buildtype') == 'release'
+  add_project_arguments(release_args, language: 'c')
+  add_project_link_arguments(release_args, language: 'c')
+endif
+
+extra_src = []
+extra_obj = []
+static_libs = false
+
+if target_machine.system() == 'windows'
+  windres = find_program('windres')
+  run_command(windres, 'src/win32/resource.rc', 'src/win32/resource.o')
+  extra_src = [
+    'src/win32/gui_windows.c',
+  ]
+  extra_obj = [
+    'src/win32/resource.o',
+  ]
```

Comment:
I think it will be a much, much, much better idea to use our builtin support: https://mesonbuild.com/Windows-module.html

```meson
if host_machine.system() == 'windows'
    winmod = import('windows')
    extra_src = ['src/win32/gui_windows.c', winmod.compile_resources('src/win32/resource.rc')]
endif
```